src/V3Number.h

```diff
@@ -24,9 +24,12 @@
 #include "V3Hash.h"
 
 #include <algorithm>
+#include <array>
 #include <cmath>
 #include <limits>
 #include <vector>
+#include <memory>
+#include <cstdlib>
```

Comment:
Sort include headers.

Reply:
I have configured clang-format so `make format` does this automatically now.

Reply:
Fixed